trillian/integration/hammer.go

```diff
@@ -77,6 +77,17 @@ func (e errSkip) Error() string {
 	return "test operation skipped"
 }
 
+// Limiter is an interface to allow different rate limiters to be used with the
```

Comment:
At the risk of being slightly more complicated, another pattern could be a channel where successfully taking an item allows you to perform an op, and you block otherwise, then there's a thing which drops as many ops/s in as is permitted.

Just thinking that this would also work for setups where you had multiple workers needing to share a global rate limit and so might be a bit more reusable elsewhere?

Feel free to take it or leave it :)

Reply:
I prefer to stick with the current approach, because it's borrowed from the fixchain stuff and seems to work OK there.